app/livechat/server/hooks/externalMessage.js

```diff
@@ -33,6 +34,8 @@ callbacks.add('afterSaveMessage', function(message, room) {
 		return message;
 	}
 
+	message = normalizeMessageAttachments(message);
```

Comment:
Do you mind of checking the `message.file` before returning? I think we should always return the `message` instead of the `normalizeMessageAttachments`.